ci/azure-pipelines-docs-integration.yml

```diff
@@ -116,12 +116,16 @@ stages:
 
     - job: test_docs_mysql
       timeoutInMinutes: 30
-      condition: or(
-          eq(stageDependencies.scope_check.changes.outputs['CheckDocsDependenciesChanges.DocsDependenciesChanged'], true),
-          eq(stageDependencies.scope_check.changes.outputs['CheckDocsChanges.DocsChanged'], true),
-          eq(variables.isDevelop, true),
-          eq(variables.isManual, true)
-        )
+      condition: false
```

Comment:
I noticed in `ci/azure-pipelines-dev.yml` that there are a few additional steps to get a mysql database up and running. Maybe this stage will work if those are added? Specifically:

```
	          - script: |
	              printf 'Waiting for MySQL database to accept connections'
	              until mysql --host=localhost --protocol=TCP --port=3306 --user=root --password='' --execute "SHOW DATABASES"; do
	                printf '.'
	                sleep 1;
	              done;
	            displayName: Wait for database to initialise
	          - script: |
	              echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" > mysql_setup_script.sql
	              mysql --host=localhost --protocol=TCP --port=3306 --user=root --password='' --reconnect < mysql_setup_script.sql
	            displayName: 'Configure mysql'
```

Reply:
Scratch that, looks like it is an issue with mysql and errors when running `DROP TABLE IF EXISTS my_table;` when the table doesn't exist (which should be fine). Take a look at https://github.com/great-expectations/great_expectations/pull/7691 for a potential solution (the python script: https://github.com/great-expectations/great_expectations/pull/7691/files#diff-cd49bca7f40d4013bc899434aa7bb1e8db522c82d9c149fdb788a4104f3bf680, specifically the `drop_existing_table` param)

Reply:
@kenwade4 With the above fixes do you think we can reenable this stage?

Reply:
The above fixes of adding the mysql setup scripts are in but they will fail with the example error in line 121.

Reply:
The stuff in the script can be attempted on the new branch that undoes the test skipping